### PR TITLE
Fix an npe in declareLegacyNamespacesAliases().

### PR DESCRIPTION
**Pull request by LucasSloan**

### CODE REVIEW

File: src/test/java/com/google/javascript/clutz/partial/optional_export_object_property.d.ts [hunk 1]
```diff
@@ -5,5 +5,3 @@ declare module 'goog:optional.export.object.property' {
   import alias = ಠ_ಠ.clutz.optional.export.object.property;
   export = alias;
 }
-declare namespace ಠ_ಠ.clutz {
```
Comment (rkirov) [inline src/test/java/com/google/javascript/clutz/partial/optional_export_object_property.d.ts]: what happened here?

Comment (LucasSloan) [reply src/test/java/com/google/javascript/clutz/partial/optional_export_object_property.d.ts]: I moved the lines that created the namespace block inside the if statement.

**Comment by LucasSloan** [comment]
Added a test, PTAL

File: src/test/java/com/google/javascript/clutz/partial/declare_legacy_namespace_goog_define.d.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+declare namespace ಠ_ಠ.clutz.legacy.namespace.goog {
```
Comment (rkirov) [inline src/test/java/com/google/javascript/clutz/partial/declare_legacy_namespace_goog_define.d.ts]: Ops, we probably should be emitting this, add '//!! TODO'

**Pull request merged**